app/about-us/about-us.css

```diff
@@ -0,0 +1,27 @@
+@media only screen and (max-width:1025px) {
+    .first-image {
+        height: 400px
+    }
+
+    .second-image {
```

Comment:
if something is on the same css property do not create multiple classes like for the second and third-image @PuniaMohit